lib/network/kmip/error-comparison.jsonc

```diff
@@ -34,7 +34,9 @@
                 "resultMessage": [
                     "[NCERRInsufficientPermissions]: ",
                     // TLS session idle timeout (24h) - needs a new connection or no idle
-                    "[NCERRUnauthorizedAccess]: Token has been revoked"
+                    "[NCERRUnauthorizedAccess]: Token has been revoked",
+                    // write after receiving above token revoked error
+                    "[NCERRUnauthorizedAccess]: Invalid token"
```

Comment:
for my own edification and as this is different from S3, worth commenting documentation link for these errors

Reply:
no documentation link, it's only something we've been told (that's why we kept an healthcheck running regularly to avoid a 24h session revoked in prod).

I had to ask thales to tweak my thales lab to reduce the session idle timeout to 10m to be able to test this

Reply:
told by whom
We should add that context to the code(without customer name ofcourse)

Reply:
idk by thales when we saw that error